client/www/index.html

```diff
@@ -69,6 +69,7 @@
 <script src="./scripts/vendor/react/react-with-addons.js"></script>
 
 <!-- d3 -->
+<script src="./scripts/modules/tracing/tracing.viz.module.js"></script>
```

Comment:
why is this file moved here?  It depends on d3 so should be loaded after d3.js.  Plus it should be with the tracing code.